core/src/main/resources/hudson/logging/LogRecorder/delete.jelly

```diff
@@ -31,9 +31,9 @@ THE SOFTWARE.
     <st:include page="sidepanel.jelly" />
     <l:main-panel>
       <form method="post" action="doDelete">
-        ${%Are you sure about deleting this log recorder?}
+        ${%delete.logrecorder(it.displayName)}
```

Comment:
Unfortunately there are ~10 translations with the "very nice" key: `Are\ you\ sure\ about\ deleting\ this\ log\ recorder?`. So this improvement for english will result in quality decrease for all the other languages.

Reply:
https://issues.jenkins-ci.org/browse/JENKINS-58157